app/models/concerns/downloadable_files/image.rb

```diff
@@ -21,6 +21,14 @@ def remove_file=(value)
       @remove_file = !value.to_i.zero?
     end
 
+    def padded_image400x200
```

Comment:
Since `Image` is a module that can be used wherever, I thought it might be a good idea if this method name was descriptive and didn't assume much with regards to `Facility`

Reply:
I might vote for `padded_image(width: 400, height: 200, background: "rgb(231, 231, 231)")`